_inc/lib/tracks/class.tracks-client.php

```diff
@@ -1,5 +1,5 @@
 <?php
 /**
- * Deprecated file since 7.5 - Jetpack_Tracks_Client is now autoloaded from 'vendor/automattic/jetpack-tracking/legacy/class.tracks-client.php'
+ * Deprecated since 7.5 – Tracks_Client are autoloaded from 'vendor/automattic/jetpack-tracking/legacy/class.tracks-client.php'
  */
-_deprecated_file( basename( __FILE__ ), 'jetpack-7.5', 'vendor/automattic/jetpack-tracking/legacy/class.tracks-client.php' );
```

Comment:
The reason I'm pointing to vendor, is because we don't ship `packages/`. These notices are going to be shown to people running production Jetpack, and `packages/` does not exist for them.